### PR TITLE
Faq

### PR DESCRIPTION
**Pull request by RickRilled**
Added a FAQ fragment in the settings activity, including our in-app help documentation.

### CODE REVIEW

**Comment by OneTrueAsian** [comment]
Also, you should merge with master to resolve that conflict git is not happy about.

File: app/src/main/res/layout/faq.xml [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            >
+
```
[the remark this answers is not in the document]
Comment (RickRilled) [reply app/src/main/res/layout/faq.xml]: Good eye, committing the changes now

[the remark this answers is not in the document]
Comment (RickRilled) [reply app/src/main/res/layout/faq.xml]: That's what I wanted the spaces and titles to indicate, but to be honest I'm not sure what you're referring to.

File: app/src/main/java/edu/calvin/cs262/cs262d/eventconnect/views/SettingsActivity.java [hunk 2]
```diff
@@ -438,4 +448,43 @@ public boolean onOptionsItemSelected(MenuItem item) {
             return super.onOptionsItemSelected(item);
         }
     }
+
+    public static class FAQFragment extends DialogFragment {
+        /**
+        TODO doc
```
[the remark this answers is not in the document]
Comment (RickRilled) [reply app/src/main/java/edu/calvin/cs262/cs262d/eventconnect/views/SettingsActivity.java]: Good eye, committing changes now

**Pull request merged**